test/modules/core/lib/composite-layer.spec.js

```diff
@@ -81,3 +104,82 @@ test('CompositeLayer#getSubLayerProps', t => {
 
   t.end();
 });
+
+test.only('CompositeLayer#getSubLayerProps(override)', t => {
```

Comment:
Keep an eye on the coverage please.

Reply:
Oops, my bad. I wonder we can add lint rule for this. Will update.